### ISSUE TITLE
Translation to Romanian

### ISSUE THREAD

**Issue by MadalinTat** [huggingface/course#590]
Hi there 👋

Let's translate the course to Romanian so that the whole community can benefit from this resource 🌎!

Below are the chapters and files that need translating - let us know here if you'd like to translate any and we'll add your name to the list. Once you're finished, open a pull request and tag this issue by including `#issue-number` in the description, where `issue-number` is the number of this issue.

> 🙋 If you'd like others to help you with the translation, you can also post in our [forums](https://discuss.huggingface.co/c/course/20) or tag [@_lewtun](https://twitter.com/_lewtun) on Twitter to gain some visibility.

## Chapters

**0 - Setup**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter0/1.mdx)

**1 - Transformer models**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/8.mdx)
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/9.mdx)
- [ ] [`10.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/10.mdx)

**2 - Using 🤗 Transformers**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter2/8.mdx)

**3 - Fine-tuning a pretrained model**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3.mdx)
- [ ] [`3_tf.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/3_tf.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter3/6.mdx)

**4 - Sharing models and tokenizers**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter4/6.mdx)

**5 - The 🤗 Datasets library**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter5/8.mdx)

**6 -  The 🤗 Tokenizers library**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/3.mdx)
- [ ] [`3b.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/3b.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/8.mdx)
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/9.mdx)
- [ ] [`10.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter6/10.mdx)

**7 - Main NLP tasks**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/4.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/7.mdx)
- [ ] [`8.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/8.mdx)
- [ ] [`9.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter7/9.mdx)

**8 - How to ask for help**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/1.mdx)
- [ ] [`2.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/2.mdx)
- [ ] [`3.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/3.mdx)
- [ ] [`4.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/4.mdx)
- [ ] [`4_tf.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/4_tf.mdx)
- [ ] [`5.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/5.mdx)
- [ ] [`6.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/6.mdx)
- [ ] [`7.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/chapter8/7.mdx)

**Events**
- [ ] [`1.mdx`](https://github.com/huggingface/course/blob/main/chapters/en/event/1.mdx)

**Comment by MadalinTat** [comment]
I want to translate chapter 0: Setup and then work on chapter 1: Transformer models. As I am translating I will try to bring people along to help me translate the rest of the chapters, but I will do as much as I can until then!

**Comment by akshayamadhuri** [comment]
@MadalinTat let me help you, though I am not native

**Comment by valentinobianco** [comment]
@MadalinTat 
I can help with the RO locale, native here. 
Regards, 
V.

**Comment by MadalinTat** [comment]
> @MadalinTat let me help you, though I am not native

Sure thing! I can review your drafts to make sure they are translated good :)

**Comment by MadalinTat** [comment]
> @MadalinTat 
> I can help with the RO locale, native here. 
> Regards, 
> V. 

Great! This week I am going to start working on it

**Comment by MadalinTat** [comment]
I am going to take care of the Chapters 1, 2 and 3 because I just finished them and it would help me reinforce my knowledge if that's alright with everyone

**Comment by stefantaga24** [comment]
Hello! I would like to know if anyone is still working on this? If yes, maybe I can chip in?

**Comment by MadalinTat** [comment]
> Hello! I would like to know if anyone is still working on this? If yes, maybe I can chip in?

Definitely you can chip in! :) We postponed it but soon we are going to start it. I will be working on the first 2 chapters. Feel free to choose any of them:)

**Comment by eduard-balamatiuc** [comment]
Hey!
We are a native here! We have a community of people with knowledge in the domain and wanted to know if there was any progress on this issue so far and if maybe we can help as well?

**Comment by eduard-balamatiuc** [comment]
> Heya👋
> 
> I would like to translate the 0-setup 1mdx into Romanian. I am a native speaker. How can I start my contribution?
> 
> Thanks!

Hey could you help us with reviewing some of the content? Basically we are almost done with translating the entire content in romanian but the only thing left is to review it, we did it together in our team at Sigmoid (and NGO from Moldova), would it be okay if maybe I create the Pull Request and you have a look there if you've got some time?

**Comment by eduard-balamatiuc** [comment]
As discussed [here](https://github.com/huggingface/course/pull/864) you can find the PR, I saw that the build failed so I will try to address that element and also to add the other PRs till the end of the week but till then feel free to have a look there and provide an initial iteration feedback @hbkdesign

**Comment by hbkdesign** [comment]
Hey @eduard-balamatiuc 👋

I've been reviewing the Romanian translation as part of my university open-source contribution project. I’ve left inline comments and submitted general feedback using the “Approve” button on the files I’ve reviewed so far.

Just wanted to double-check: am I following the correct review process here? Let me know if there's anything I should do differently or if there's a preferred way of submitting reviews in this repo. 

Thanks!

**Comment by eduard-balamatiuc** [comment]
Hey @hbkdesign,
From my knowledge you should approve the file or the PR only at the point when you don't think any changes have to be made additionally or any changes have to be done, other than that, you can just leave comments until those will be solved by the assignee of the PR. 

Thank you for the review btw, I will do my best to address the threads as soon as possible!

**Comment by hbkdesign** [comment]
Thanks for clarifying that, @eduard-balamatiuc.
That makes perfect sense - I’ll make sure to only approve when no further suggestions are needed and otherwise leave comments for now.

You're very welcome - happy to help! 
Let me know if anything I suggested needs clarification or if there's anything I can assist with further.

**Comment by hbkdesign** [comment]
Hi @eduard-balamatiuc, 

I finished reviewing the Romanian translation of Chapter 1. I can contribute by reviewing Chapter 2 as well, please let me know if you open a PR for that.
